README.md

```diff
@@ -75,7 +75,7 @@ qse clear
 1. Run the following command in the terminal to install the required `node` packages:
 
 ```bash
-    npm i -g @cse-25/qse
+   npm i -g quick_start_express
 ```
```

Comment:
The extra space in the front of the command is also copied when we click the GitHub copy icon. Tab space the entire block like the below commands to avoid extra spaces being copied. 


```suggestion
    ```bash
    npm i -g quick_start_express
    ```
```